airbyte-cdk/bulk/toolkits/load-low-code/src/main/kotlin/io/airbyte/cdk/load/discoverer/operation/Property.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.discoverer.operation
+
+import com.fasterxml.jackson.databind.JsonNode
+import io.airbyte.cdk.load.data.AirbyteType
+import io.airbyte.cdk.load.data.FieldType
+import java.util.function.Predicate
+
+class Property(
```

Comment:
Similar comment as for most objects, they need to be documented. It's not clear what the property refers to. In my mind, this is a Property of the DestinationObject. Hovewer, we're in the operation package so know I am confused.